docs/reference/mapping/types/sparse-scored-terms.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="xpack"]
+[[sparse-scored-terms]]
+=== Sparse Scored Terms field type
+++++
+<titleabbrev>Sparse Scored Terms</titleabbrev>
+++++
+
+A field to store terms along with their associated scores.
+This data is defined using two paired arrays:
+
+* A `terms` array of string values indicating which terms to be indexed.
+* A corresponding `scores` array of positive <<number, `integer`>> numbers indicating
+the individual term score. Consider this value similar to term frequency.
+
+Because the elements in the `terms` array correspond to the elements in the
+same position of the `scores` array, these two arrays must have the same length.
```

Comment:
Might be worth saying something here about the fact we don't store any position information, so phrase or interval queries won't work?